src/System.Memory/tests/Span/Reverse.cs

```diff
@@ -0,0 +1,331 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using System.Runtime.CompilerServices;
+using static System.TestHelpers;
+
+namespace System.SpanTests
+{
+    public static partial class SpanTests
+    {
+        [Fact]
+        public static void ReverseEmpty()
+        {
+            var span = Span<byte>.Empty;
+            span.Reverse();
+
+            byte[] actual = { 1, 2, 3, 4 };
+            byte[] expected = { 1, 2, 3, 4 };
+
+            span = actual;
+            span.Slice(2, 0).Reverse();
+            
+            Assert.Equal<byte>(expected, span.ToArray());
```

Comment:
This is not a test of a emot span

Reply:
It is testing that reversing an empty slice of a span does not reverse anything in the backing array.